src/PAModel/CanvasDocument.cs

```diff
@@ -96,7 +96,7 @@ public class CanvasDocument
         /// </summary>
         /// <param name="fullPathToMsApp">path to an .msapp file</param>
         /// <returns>A tuple of the document and errors and warnings. If there are errors, the document is null.  </returns>
-        public static (CanvasDocument, ErrorContainer) LoadFromMsapp(string fullPathToMsApp)
+        public static (CanvasDocument, ErrorContainer) LoadFromMsapp(string fullPathToMsApp, bool throwOnException = false)
```

Comment:
This seems odd, why is it an option for the caller? If the caller doesn't want the exceptions, it can always wrap this in a 
`try {} catch { /* do nothing */ }`